### ISSUE TITLE
Select UI theme colors compliant with WCAG 2.1 standard

### ISSUE THREAD

**Issue by wk-y** [LunariumGame/Lunarium#95]
The minimum standard to meet is `WCAG AA`. `WCAG AAA` is not required. "Graphical Objects and User Interface Components" should be used for selecting our UI border colors.

This tool can be used to validate the color contrast of two colors: <https://webaim.org/resources/contrastchecker/>
